useful_resources/computer_setup/terminal_app_setup.md

```diff
@@ -0,0 +1,47 @@
+# Setting Up Your Terminal
+
+
+## Overview
+
+These are some ways to make your terminal app more informative.
+
+
+## Create A .bash_profile File
+
+### What is a .bash_profile file?
+
+This is often referred to as a _configuration script_. It can contain variable
+declarations, export variables, and commands to be executed when you start up
+your Terminal[*][ref1]. One common use case is store your API keys and tokens so
+they are not hard-coded in your scripts.
```

Comment:
Which you should never do! All Parsons classes that require credentials allow you to set environmental variables, so you can just set these once and then forget about it.